src/log.py

```diff
@@ -0,0 +1,55 @@
+"""
+Data Log
+
+By: Norma Chang
+"""
+import json
+import time
+
+def log(data, data_type):
+    """
+    Logs dictionary data from leader
```

Comment:
Invalid docstring function format.

Reply:
Please review any other function in the repository.

Reply:
updated docstring